### ISSUE TITLE
Training set+Validation set+Test set != total data rows

### ISSUE THREAD

**Issue by GPUSA** [uber/ludwig#258]
Hello,
My training data set size is 82278 rows. But when I run a train on the csv file, I see below size for my train, validation and test sets. Can you help me run training on the full data set?

Loading NLP pipeline
Writing dataset
Writing train set metadata with vocabulary
Training set: 21343
Validation set: 2988
Test set: 6041

**To Reproduce**

Please provide code, yaml definition file and a sample of data in order to entirely reproduce the issue.
Issues that are not reproducible will be ignored.

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
- Python version
- Ludwig version

**Additional context**

**Comment by w4nderlust** [comment]
@GPUSA please provide a reproducible example, as already requested in the form for submitting an issue.

**Comment by GPUSA** [comment]
@w4nderlust thanks for the response. I cannot share my current dataset. I am trying to find another open dataset that I can reproduce the issue with.

**Comment by w4nderlust** [comment]
You can obfuscate it and remove all features but one. Or you reproduce the same structure using the dataset synthesyze in the \/data7\ directory.

**Comment by GPUSA** [comment]
It appears to be a formatting issue with my csv file. I generated a new file and was able to run the train on the complete dataset. Thanks @w4nderlust
